src/topic_training/src/Demo.py

```diff
@@ -0,0 +1,92 @@
+''' Import rospy to use ROS with python. '''
+import rospy
+''' Import the message type we will be publishing. '''
+from std_msgs.msg import String
+
+''' Constant variables to be used below. '''
+QUEUE_SIZE = 10
+RATE = 10
+
+'''
+This function will get called when a new message has 
+arrived on the topic.
+'''
+def callback(data):
```

Comment:
Type-annotations would help here, especially for the type of `data` and return type of the function.
```suggestion
def callback(data: String) -> None:
```

Reply:
Fixed in a4b953f.

Reply:
I would keep the `std_msgs` package prefix in this type annotation since it makes clearer that the parameter is a ROS message and not a normal string.

Reply:
Fixed in 211a686.